nautobot/project-static/css/dark.css

```diff
@@ -190,6 +190,12 @@ html[data-theme="dark"] .navbar-nav>li>a:hover {          /* Navbar top level me
   color: rgb(224, 120, 7) !important;
 }
 
+html[data-theme="dark"] .dropdown-menu button:hover,      /* Navbar button item when hovering */
+html[data-theme="dark"] .dropdown-menu>li>button:hover {
+  color: rgb(51, 51, 51) !important;
+  background-color: rgb(196, 204, 210) !important;
```

Comment:
Curious why a dropdown `button` needs a custom background-color but a dropdown `a` (below) doesn't?

Reply:
The default behavior is to invert the colors that are on the light theme. The issue with buttons, is the light theme displays them as dark, this is likely to match the nav-bar buttons. Inverting them changes them to white, which is not what we want in a dark theme. This overrides the background-color so that it stays the dark grey color.

As to why it is not needed on the dropdown `a`, the light theme displays the `a` with a white background, so inverting is the correct behavior.